### ISSUE TITLE
创建临时表，然后执行cluter 临时表，报表不存在

### ISSUE THREAD

**Issue by mass3** [ADBSQL/AntDB#34]
### 创建临时表，然后执行cluter 临时表，报表不存在
postgres=# create temp table clstr_temp (col1 int primary key, col2 text);
CREATE TABLE
postgres=# insert into clstr_temp values (2, 'two'), (1, 'one');
INSERT 0 2
**postgres=# cluster clstr_temp using clstr_temp_pkey;
ERROR:  relation "clstr_temp" does not exist**
postgres=# \d+ clstr_temp
                     Table "pg_temp_2.clstr_temp"
 Column |  Type   | Modifiers | Storage  | Stats target | Description 
--------+---------+-----------+----------+--------------+-------------
 col1   | integer | not null  | plain    |              | 
 col2   | text    |           | extended |              | 
Indexes:
    "clstr_temp_pkey" PRIMARY KEY, btree (col1)

**单pg现象:**
postgres=# create temp table clstr_temp (col1 int primary key, col2 text);
insert into clstr_temp values (2, 'two'), (1, 'one');
CREATE TABLE
postgres=# insert into clstr_temp values (2, 'two'), (1, 'one');
cluster clstr_temp using clstr_temp_pkey;
INSERT 0 2
postgres=# cluster clstr_temp using clstr_temp_pkey;
CLUSTER
postgres=# 
### 测试sql:
```
create temp table clstr_temp (col1 int primary key, col2 text);
insert into clstr_temp values (2, 'two'), (1, 'one');
cluster clstr_temp using clstr_temp_pkey;
```

**Comment by hsiaoby** [comment]
fixed by cc0147a0625054884675067042b16a8b6fe3ac20

**Comment by mass3** [comment]
cluster成功
```
postgres=# create temp table clstr_temp (col1 int primary key, col2 text);
insert into clstr_temp values (2, 'two'), (1, 'one');
CREATE TABLE
postgres=# insert into clstr_temp values (2, 'two'), (1, 'one');
cluster clstr_temp using clstr_temp_pkey;
INSERT 0 2
postgres=# cluster clstr_temp using clstr_temp_pkey;
CLUSTER
postgres=#
```
